sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning_resnet.ipynb

```diff
@@ -524,7 +524,7 @@
    "source": [
     "### Run iterative model pruning\n",
     "\n",
-    "After having gone through the code step by step, we are ready to run the full worfklow. The following cell runs 10 pruning iterations: in each iteration of the pruning a new SageMaker training job is started, where it emits gradients and activation outputs to Amazon S3. Once the job has finished, filter ranks are computed and the 100 smallest filters are removed.\n",
+    "After having gone through the code step by step, we are ready to run the full worfklow. The following cell runs 1 pruning iteration: in each iteration a new SageMaker training job is started, where it emits gradients and activation outputs to Amazon S3. Once the job has finished, filter ranks are computed and the 100 smallest filters are removed.\n",
```

Comment:
The following cell runs 1 pruning iteration for a tutorial purpose. Change the range of the for loop to 10 to replicate the same result shown in the [Pruning machine learning models with Amazon SageMaker Debugger and Amazon SageMaker Experiments](https://aws.amazon.com/blogs/machine-learning/pruning-machine-learning-models-with-amazon-sagemaker-debugger-and-amazon-sagemaker-experiments/) blog and the figure below the cell. In each iteration, SageMaker starts a new training job with a pruned model, and Debugger collects gradients and activation outputs to Amazon S3.